### DISCUSSION TITLE
Language Bar as well as most used language Disappears after a commit

### DISCUSSION THREAD

**Discussion by arshb22** [github-linguist/linguist#5926] [Q&A] [answered]
Hello,

I keep having a problem with linguist wherein upon pushing a file [really depends - sometime after 10] the language bar disappears and the most used language no longer appears on my repository. I've tested the files individually by pushing them to a test repo and seeing what linguist says and it looks like it can detect the languages. However, when I decide to push the files one by one it just seems to break at some point [it was the same when I pushed them in bulk]. Here's a link to my repository: https://github.com/arshb22/Virtual-Reality-and-Game-Design 

I've also waited quite a while for something to change, but nothing has changed as of late.

**Comment by lildude** [comment, marked as answer]
You're not seeing any analysis because your repo has a lot of files in it so it hitting [this hard limit](https://github.com/github/linguist/blob/c497388e7f88736d7a2fe2735f64366b41a4b5f8/lib/linguist/repository.rb#L132).

You'll need to reduce the number of files in your repo if you want langauge analysis to happen.

This is the same as discussed in https://github.com/github/linguist/discussions/5878, https://github.com/github/linguist/discussions/5538, https://github.com/github/linguist/discussions/5445 and https://github.com/github/linguist/discussions/5524
